student-projects/דניאל פוז'נקו/avg.ts

```diff
@@ -0,0 +1,16 @@
+let sumNumbers = 0;
+let numberCount = 0;
+let userInput = Number(prompt("Enter a number"));
+
+while (userInput !== null) {
```

Comment:
Since the input is immediately cast to `number`, this condition would always be true and we get an infinite loop.